rmw_opendds_shared_cpp/include/rmw_opendds_shared_cpp/qos.hpp

```diff
@@ -28,14 +28,14 @@
 RMW_OPENDDS_SHARED_CPP_PUBLIC
 bool
 get_datareader_qos(
-  DDS::DomainParticipant * participant,
+  DDS::Subscriber * subscriber,
```

Comment:
`_var` version of this?

Reply:
We don't necessarily have to use `_var` everywhere.  Especially if you're migrating existing code it can be difficult to see what has to change.  So I guess my suggestion would be to use it where it's helpful (and especially for new code), but we don't have to refactor to it everywhere.

Reply:
I think that's a reasonable approach that fits this project well.